cloudinit/config/cc_disk_setup.py

```diff
@@ -133,8 +133,11 @@ def handle(_name, cfg, cloud, log, _args):
             if not isinstance(definition, dict):
                 log.warning("Invalid disk definition for %s" % disk)
                 continue
-
             try:
+                if definition.get("timeout"):
```

Comment:
I suspect `definition.get("timeout")` is a string here, and `wait_for_files` treats its `maxwait` parameter as a number. Do we need to convert this to an `int`?

Reply:
Good catch thanks